lib/components/TriviaComponent.dart

```diff
@@ -41,13 +41,7 @@ class _TriviaComponentState extends State<TriviaComponent> {
       builder: (BuildContext context) {
         return AlertDialog(
           title: const Text('🎉 Good Job!'),
-          content: const SingleChildScrollView(
-            child: ListBody(
-              children: <Widget>[
-                Text('You got the correct answer.'),
-              ],
-            ),
-          ),
```

Comment:
None of this was necessary

Reply:
I don't remember why I even added that...